### PR TITLE
MS. Laud Misc. 159: new description

### PR DESCRIPTION
**Pull request by adunning**

### CODE REVIEW

File: collections/Laud_Misc/MS_Laud_Misc_159.xml [hunk 1]
```diff
@@ -38,31 +37,154 @@
...
-                        <title>Bible, Octateuch</title>
-                     </bibl>
-                     <textLang mainLang="la">Latin</textLang>
+                     <title key="work_4982">Commentarium in Octateuchum</title>
+                     <rubric>Incipit liber questionum super genesis ex dictis sanctorum patrum agustini· gregorii· hieronimi· ambrosii· hilarii· hisidori· eucherii· et iunilli·</rubric>
+                     <msItem>
+                        <locus>(fols. 29r–97r)</locus>
+                        <title>In Genesin</title>
+                        <incipit><add place="margin">Δ.</add> Primo omnium preceptor mi⹎ inquiri mihi necesse est, quis huius libri qui Genesis apud nos dicitur scribtor habeatur. <add place="margin">M.</add> Sicut a maioribus</incipit>
+                        <explicit>Dicente domino. Erit in nouissimis diebus effundande spiritu meo suer omnem carnem.</explicit>
```
Comment (holfordm) [inline collections/Laud_Misc/MS_Laud_Misc_159.xml:76]: typo? (super?)

Comment (adunning) [reply collections/Laud_Misc/MS_Laud_Misc_159.xml]: Well caught!

File: collections/Laud_Misc/MS_Laud_Misc_159.xml [hunk 2]
```diff
@@ -38,31 +37,154 @@
...
+                        <extent>i + 137 + i leaves <dimensions type="leaf" unit="mm">
+                              <height>365</height>
+                              <width>275</width>
+                           </dimensions>
+                        </extent>
+                        <foliation>Modern foliation.</foliation>
+                        <collation>Mostly in quaternions. First leaf (with title page) lost; one quire lost after after fol. 39. <signatures>Quires signed on the final page: <q>I</q> (fol. 7v), <q>Q II</q> (15v), <q>III</q> (fol. 23v), <q>Q IIII–V</q> (fol. 31v, fol. 39v), <q>VII–VIII</q> (fols. 47v, 55v, 63v).</signatures></collation>
                      </supportDesc>
+                     <layoutDesc>
+                        <layout columns="2" writtenLines="36">Ruled in ink. 2 columns of 36 lines, ruled space <dimensions type="ruled" unit="mm">
```
Comment (holfordm) [inline collections/Laud_Misc/MS_Laud_Misc_159.xml:158]: is this right? (ink ruling). seems so unusual at this date I thought I should check!

Comment (adunning) [reply collections/Laud_Misc/MS_Laud_Misc_159.xml]: That was a slip – thank you!

**Pull request merged**